test/lib/jdk/test/whitebox/WhiteBox.java

```diff
@@ -556,6 +558,31 @@ public void clearInlineCaches(boolean preserve_static_stubs) {
   // Force Full GC
   public native void fullGC();
 
+  private static Method wfrp = null;
+  /* Wait for reference processing, via Reference.waitForReferenceProcessing().
+   * Callers of this method will need the
+   * @modules java.base/java.lang.ref:open
+   * jtreg tag.
+   *
+   * This method should usually be called after a call to WhiteBox.fullGC().
+   */
+  public static void waitForReferenceProcessing() {
+    try {
+      if (wfrp == null) {
+        Class refClass = Class.forName("java.lang.ref.Reference");
+        Method[] methods = refClass.getDeclaredMethods();
+        wfrp = Arrays.stream(methods).filter((m) -> m.getName().equals("waitForReferenceProcessing")).findFirst().get();
```

Comment:
Use `getDeclaredMethod`, as previously suggested by @RogerRiggs in comment for the copy in ForceGC?

Reply:
Yep